components/Dialogs/tests/snapshot/MDCAlertControllerActionsTests.m

```diff
@@ -407,11 +450,22 @@ - (void)addFirstLongActionWithEmphasis:(MDCActionEmphasis)emphasis {
                                                           handler:nil]];
 }
 
-- (void)addShortActionWithEmphasis:(MDCActionEmphasis)emphasis cancelAction:(BOOL)cancelAction {
-  NSString *title = cancelAction ? @"Cancel" : @"OK";
-  [self.alertController addAction:[MDCAlertAction actionWithTitle:title
+- (void)addOKActionWithEmphasis:(MDCActionEmphasis)emphasis {
+  [self.alertController addAction:[MDCAlertAction actionWithTitle:@"OK"
                                                          emphasis:emphasis
                                                           handler:nil]];
 }
 
+- (void)addCancelActionWithEmphasis:(MDCActionEmphasis)emphasis {
+  [self.alertController addAction:[MDCAlertAction actionWithTitle:@"Cancel"
+                                                         emphasis:emphasis
+                                                          handler:nil]];
+}
+
+- (void)addLowEmphasisAction:(NSString *)action {
+  [self.alertController addAction:[MDCAlertAction actionWithTitle:action
+                                                         emphasis:MDCActionEmphasisLow
+                                                          handler:nil]];
+}
+
```

Comment:
Please pull the refactoring out to a separate PR and then land the new snapshot test independently.